diff --git a/vendor/zendframework/zendframework/library/Zend/Mvc/Service/ViewHelperManagerFactory.php b/vendor/zendframework/zendframework
/library/Zend/Mvc/Service/ViewHelperManagerFactory.php
index 30757df..99ef575 100644
--- a/vendor/zendframework/zendframework/library/Zend/Mvc/Service/ViewHelperManagerFactory.php
+++ b/vendor/zendframework/zendframework/library/Zend/Mvc/Service/ViewHelperManagerFactory.php
@@ -65,7 +65,7 @@ class ViewHelperManagerFactory extends AbstractPluginManagerFactory
             $router = Console::isConsole() ? 'HttpRouter' : 'Router';
             $helper->setRouter($serviceLocator->get($router));

-            $match = $serviceLocator->get('application')
+            $match = $serviceLocator->get('thing')
                 ->getMvcEvent()
                 ->getRouteMatch()
             ;

diff --git a/vendor/zendframework/zend-developer-tools/Module.php b/vendor/zendframework/zend-developer-tools/Module.php
index def8572..b531741 100644
--- a/vendor/zendframework/zend-developer-tools/Module.php
+++ b/vendor/zendframework/zend-developer-tools/Module.php
@@ -184,7 +184,7 @@ class Module implements
                 'ZendDeveloperTools\Event' => function ($sm) {
                     $event = new ProfilerEvent();
                     $event->setReport($sm->get('ZendDeveloperTools\Report'));
-                    $event->setApplication($sm->get('Application'));
+                    $event->setApplication($sm->get('thing'));

                     return $event;
                 },
